html/pfappserver/root/static.alt/src/components/pfFormPassword.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+  <b-form-group horizontal :label-cols="(columnLabel) ? labelCols : 0" :label="$t(columnLabel)"
+    :state="isValid()" :invalid-feedback="getInvalidFeedback()"
+    class="pf-form-password" :class="{ 'mb-0': !columnLabel, 'is-focus': focus }">
+    <b-input-group class="input-group-password">
+      <b-form-input
+        v-model="inputValue"
+        v-bind="$attrs"
+        ref="input"
+        :type="type"
+        :state="isValid()"
+        @input.native="validate()"
+        @keyup.native="resetTest($event)"
+        @change.native="onChange($event)"
+        @focus.native="focus = true"
+        @blur.native="focus = false"
+      >
+      </b-form-input>
+      <b-input-group-append>
+        <b-button-group v-if="test" rel="testResultGroup">
+          <b-button v-if="testResult !== null" variant="light" disabled tabindex="-1">
+            <span class="mr-1" :class="{ 'text-danger': !testResult, 'text-success': testResult }">{{ testMessage }}</span>
+          </b-button>
+        </b-button-group>
+        <b-button-group rel="prefixButtonGroup">
+          <b-button v-if="test" class="input-group-text" @click="runTest()" :disabled="isLoading || !this.value" tabindex="-1">
+            {{ $t('Test') }}
+            <icon v-show="isTesting" name="circle-notch" spin class="ml-2 mr-1"></icon>
+            <icon v-if="testResult !== null && testResult" name="check" class="ml-2 mr-1 text-success"></icon>
+            <icon v-if="testResult !== null && !testResult" name="times" class="ml-2 mr-1 text-danger"></icon>
+          </b-button>
+          <b-button class="input-group-text" @click="click()" @mouseover="over()" @mousemove="over()" @mouseout="out()" :disabled="!this.value && this.type === 'password'" :pressed="visible" tabindex="-1"><icon name="eye"></icon></b-button>
```

Comment:
Why is it necessary to act on the "mousemove" event?

Reply:
I find it combining `over` and `move` together helps with situations when the `over` event is missed.